### ISSUE TITLE
Trying to recreate hands dataset

### ISSUE THREAD

**Issue by GodzSom** [lucidrains/stylegan2-pytorch#90]
Hi! I'm trying to generate hands using the same dataset but my results are not as good. 
Can you share the training config? 
Thanks!

**Comment by lucidrains** [comment]
@GodzSom hi! what resolution are you training at? try increasing the `--gradient-accumulate-every` I don't remember having any other special settings (attention, or contrastive loss reg)

**Comment by GodzSom** [comment]
@lucidrains Thank you for the reply!
The resolution is 128.
I tried increasing the --gradient-accumulate-every but the result is still not as good.
![32-ema](https://user-images.githubusercontent.com/37606324/87756155-93b11f00-c832-11ea-9e92-be90cc2e053c.jpg)

If it's not inconvenient, can you share the pre-trained model? Thanks again!

**Comment by lucidrains** [comment]
@GodzSom that's actually pretty good for 128! For even better quality, you will need to up the resolution to 256!

**Comment by lucidrains** [comment]
@GodzSom always have attention turned on if you can afford it!
